dts/bindings/sensor/bosch,bmp180.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024 Chris Ruehl
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for BMP180
+
+include: sensor-device.yaml
+
+properties:
+  int-gpios:
+    type: phandle-array
```

Comment:
I do not see any place where this is used?

Reply:
Arg, that is from the bmp280 .. but when I merge the bmp180-i2c , I will get rid of it.
Thanks for catch this.